tools/exporter/README.md

```diff
@@ -1,74 +1,99 @@
-# Audit exporter tool
-
+# Audit Exporter Tool
 Allows you to export audit data from the SafetyCulture Platform and save them anywhere on your computer.
+Supported export formats: PDF, MS WORD (docx), JSON, and CSV. Media and Web Report Link exporting is also supported.
 
-Supported export formats: PDF, MS WORD (docx), JSON, and CSV. Media exporting is also supported.
+## Installation  
+``` 
+pip install safetyculture-sdk-python
+```
 
-## Installation
+This will install
+* SafetyCulture Python SDK -- See top level [README.md](https://github.com/SafetyCulture/safetyculture-sdk-python/blob/master/README.md) for more information. 
+* iAuditor Exporter Tool
+* README files
 
-  1. First install the Python SDK (see top-level Readme)
-  2. Switch to this directory (safetyculture-sdk-python/tools/exporter)
-  3. Execute the following command from the command line:
-    ``pip install -r requirements.txt``
-  4. Edit config.yaml and replace ``YOUR_SAFETYCULTURE_API_TOKEN`` with your SafetyCulture API token
 
+## Initial Setup
+IMPORTANT: If you have used previous versions of the exporter tool, you should run it from the same folder you have run the tool from in the past.
+Otherwise, the exporter will start exporting from the earliest available audits, rather than from where the last successful export left off. 
+The export tool reads and writes to a file named `last_successful.txt` to keep track of what has already been exported. You'll find this file anywhere that you have run the exporter tool before. 
+Alternatively, you can move the `last_successful.txt` file if you prefer to export from a different location.
 
-## How to run
+If this is your first time using the exporter tool, follow these steps to get set up: 
+1. To automatically create a configuration file (which is needed to run the exporter tool), run:  
+```
+iauditor_exporter --setup
+```
+* You will be prompted for an iAuditor username and password which will be used to generate an API token. 
+Note that your credentials will not be saved in any capacity.
```

Comment:
Note that your username and password will not be saved, only used to generate the API token which is saved in the auto-generated configuration file.  
Then perhaps remove the sentence "A basic config...."